docker-compose.yml

```diff
@@ -351,6 +357,9 @@ services:
         BASE_IMAGE: sentry-self-hosted-local
     entrypoint: "/entrypoint.sh"
     command: '"0 0 * * * gosu sentry sentry cleanup --days $SENTRY_EVENT_RETENTION_DAYS"'
+  ingest-profiles:
```

Comment:
small thing, can this be moved up to be under these lines?

https://github.com/getsentry/self-hosted/blob/8aa571cb758406eccb7aefe13b3b7ebf1a34e5c2/docker-compose.yml#L332-L337

This file is pretty large and it would keep things a bit more organized